### PR TITLE
Blog post about initial filing research

### PR DESCRIPTION
**Pull request by cmaczo**
* Adds the blog post itself
* Adds the PDF file of the whole report

### CODE REVIEW

File: posts/2024/04/28/court-systems-research.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: What We Have Learned About Court Systems So Far
+date: "2024-04-29"
+tags:
+- "E-Filing"
+- "Partnerships"
+- "Court Systems"
+- "Case Management"
+author: 
+- "Michael Lissner and Colin MacArthur"
+excerpt: ”Over the last few weeks, we’ve been on a listening tour with the many admirable organizations who advance access to justice through technology, and want to improve court systems and processes. In line with our commitments to openness and transparency, we’re sharing what we have learned.”
```
Comment (mlissner) [inline posts/2024/04/28/court-systems-research.mdx:11]: Curly quotes could be part of the problem. They drive me nuts.

```suggestion
excerpt: "Over the last few weeks, we’ve been on a listening tour with the many admirable organizations who advance access to justice through technology, and want to improve court systems and processes. In line with our commitments to openness and transparency, we’re sharing what we have learned."
```

File: posts/2024/04/28/court-systems-research.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+- **Courts pay big to make small changes to systems that might help people.** Many courts and filer advocates want to change the text, flow or other details of their eFiling or case management system so it is clearer for everyone. But after the system is implemented, these changes are often very expensive. Sometimes, they even require new contracts. Even when a simple change could prevent filer mistakes, and save court timer, it often never happens.
+
+These are just a few of the findings in our report. To learn more, including a detailed overview of the way these systems fit together, read our full report.
+
+<div class="text-center"><a class="text-center whitespace-nowrap border border-transparent rounded-md shadow-sm text-base font-medium text-white text-sm no-underline bg-purple-800 hover:bg-purple-900 inline-flex px-6 py-4 text-lg" href="/pdf/Problems-Court-Filing-CMS-Perspectives-from-Advocates-and-Vendors.pdf">Read the report</a></div>
```
Comment (mlissner) [inline posts/2024/04/28/court-systems-research.mdx:26]: There's a button widget you can use. Check out example.mdx.

**Review by mlissner — CHANGES_REQUESTED** [review]

File: posts/2024/04/28/court-systems-research.mdx (hunk 2 above)
Comment (cmaczo) [reply posts/2024/04/28/court-systems-research.mdx]: Done in [8102542](https://github.com/freelawproject/free.law/pull/180/commits/81025426fa6a3dfa9f70e31afe03779be19993ca)

**Comment by cmaczo** [comment]
@mlissner Issues fixed and ready for your review again.

File: posts/2024/04/29/court-systems-research.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+- **Filer fees heavily subsidize filing systems, and that's hard on self-represented litigants.** Courts don't pay for most filing systems; the people courts serve do. Litigants have to "pay to play," and that creates an additional barrier for many self-represented applicants. Applying for a fee waiver often forces busy litigants through another set of hoops. Many of our interviewees yearned for new business models for eFiling systems that don't depend on user fees.
+
+- **Courts pay big to make small changes to systems that might help people.** Many courts and filer advocates want to change the text, flow or other details of their eFiling or case management system so it is clearer for everyone. But after the system is implemented, these changes are often very expensive. Sometimes, they even require new contracts. Even when a simple change could prevent filer mistakes, and save court timer, it often never happens.
+
+These are just a few of the findings in our report. To learn more, including a detailed overview of the way these systems fit together, read our full report.
```
Comment (mlissner) [inline posts/2024/04/29/court-systems-research.mdx:25]:
```suggestion
These are just a few of the findings in our first report on this topic. To learn more, including a detailed overview of the way these systems fit together, read our full report.
```

File: posts/2024/04/29/court-systems-research.mdx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+
+<p className="text-center">
+  <PurpleButton href="/pdf/Problems-Court-Filing-CMS-Perspectives-from-Advocates-and-Vendors.pdf" size="lg">Read the Report</PurpleButton>
+</p>
+
+**We are just getting started and we need your help!**
+Over the next few months, we want to talk more with courts to learn more about what could help them speed their operations and better serve their users. If you know an interested court or work at one, [please get in touch][contact-page].
+
+[contact-page]: /contact/
```
Comment (mlissner) [inline posts/2024/04/29/court-systems-research.mdx:35]: I think this can all go under the what's next heading, where it will make more sense?

**Review by mlissner — CHANGES_REQUESTED** [review]
Ok, I gave it one more read through and have a couple more tweaks to suggest. 

Also, can you link to it from the hiring page?

**Comment by mlissner** [comment]
@cmaczo, checking on this one: Is it ready, or do you have a few more tweaks?

**Comment by cmaczo** [comment]
@mlissner Just added the link from the hiring page you requested. I think this is ready to go.

**Pull request merged**